packages/plugins/rubygem-smart_proxy_salt/rubygem-smart_proxy_salt.spec

```diff
@@ -89,31 +130,36 @@ EOF
 
 %files
 %dir %{gem_instdir}
+%{_bindir}/foreman-node
+%config(noreplace) %attr(0640, root, foreman-proxy) %{foreman_proxy_settingsd_dir}/salt.saltfile
+%config(noreplace) %attr(0640, root, foreman-proxy) %{foreman_proxy_settingsd_dir}/salt.yml
+%license %{gem_instdir}/LICENSE
 %{gem_instdir}/bin
-%{gem_instdir}/sbin
-%{gem_instdir}/cron
-%{gem_instdir}/lib
 %{gem_instdir}/bundler.d
+%{gem_instdir}/cron
```

Comment:
Perhaps the install command should be a move so we don't ship the file twice

Reply:
Fixed

Reply:
I think this can be left out then?